features/gba-dev/Informatievragen-fields-gba.feature

```diff
@@ -0,0 +1,643 @@
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde                             |
+      | straatnaam (11.10)                         | <straat>                           |
+      | naam openbare ruimte (11.15)               | <straat>                           |
+      | huisnummer (11.20)                         | <huisnummer>                       |
+      | postcode (11.60)                           | <postcode>                         |
+      | woonplaats (11.70)                         | <woonplaats>                       |
+      | identificatiecode verblijfplaats (11.80)   | <adresseerbaarObjectIdentificatie> |
+      | identificatiecode nummeraanduiding (11.90) | <nummeraanduidingIdentificatie>    |
+      | locatiebeschrijving (12.10)                | <locatiebeschrijving>              |
+      | gemeente_code                              | 0599                               |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                              |
+      | type                | RaadpleegMetBurgerservicenummer     |
+      | burgerservicenummer | <burgerservicenummer>               |
+      | fields              | verblijfplaats.verblijfadres.<veld> |
+      Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde                             |
+      | straat                           | <straat>                           |
+      | huisnummer                       | <huisnummer>                       |
+      | postcode                         | <postcode>                         |
+      | woonplaats                       | <woonplaats>                       |
+      | locatiebeschrijving              | <locatiebeschrijving>              |
+      | adresseerbaarObjectIdentificatie | <adresseerbaarObjectIdentificatie> |
+      | nummeraanduidingIdentificatie    | <nummeraanduidingIdentificatie>    |
+	  | naamOpenbareRuimte               | <naamOpenbareRuimte>               |
+
+      Voorbeelden:
+      | burgerservicenummer | adressoort          | veld                | beschikbaarheid  | naamOpenbareRuimte | straat      | huisnummer | postcode | woonplaats | adresseerbaarObjectIdentificatie | nummeraanduidingIdentificatie | locatiebeschrijving          |
+      | 000000267           | binnenlands adres   | straat              | beschikbaar      | Boterondiep        | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
+      | 000000267           | binnenlands adres   | locatiebeschrijving | niet beschikbaar |                    | Boterondiep | 31         | 3077AW   | Rotterdam  | 0599010000208579                 | 0599200000219678              |                              |
```

Comment:
Als de locatiebeschrijving gevraagd wordt en die is niet beschikbaar waarom wordt dan de rest wel geleverd ? dan wordt er toch een lege verblijfsplaats geleverd ?